packages/credentials/src/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2018-2023, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @module @kiltprotocol/credentials
+ */
+
+export * as Attestation from './attestation/index.js'
+export * from './V1/index.js'
+export * as Presentation from './presentation/index.js'
+export * as CType from './ctype/index.js'
+export { DelegationNode, DelegationNodeUtils } from './delegation/index.js'
+export * as DataIntegrity from './proofs/DataIntegrity.js'
+
+export * as issuer from './issuer.js'
+export * as holder from './holder.js'
+export * as verifier from './verifier.js'
```

Comment:
I thought we were going to capitalise on them.

Reply:
this happens as part of https://github.com/KILTprotocol/sdk-js/pull/813

Reply:
Most excellent